workflow/rules/common.smk

```diff
@@ -1100,7 +1114,7 @@ def sample_has_umis(sample):
 
 
 def get_umi_read_structure(wildcards):
-    return "-r {}".format(
+    return "-s true -r {}".format(
         extract_unique_sample_column_value(wildcards.sample, "umi_read_structure")
     )
```

Comment:
The first suggestion has been added. The second one does not seem to work in this case as lookups are intended in input and output but also because we are currently throwing an error in case there are multiple entries in the dataframe.